content/en/apps/examples/direct-to-patient.md

```diff
@@ -0,0 +1,93 @@
+
+The CHT-RapidPro integration enables App builders to build complex conversational SMS workflows that support patient care and support coordination of care at the facility and community levels.
+
+ ### Direct-to-patient workflow technical architecture
+
+ The CHT-RapidPro integration enables App builders to build complex conversational SMS workflows that support patient care and support coordination of care at the facility and community levels.
+
+{{< figure src="technicalArchitecture.png"  link="technicalArchitecture.png" alt="Direct Messaging technical architecture" title="" class="left col-9">}}
+## Voluntary medical male circumcision app co-designed by I-TECH and Medic
+Since 2018, I-TECH and Medic have collaboratively designed and developed a CHT based voluntary medical male circumcision (VMMC) app. The app supports automated, interactive and two-way messaging between male circumcision health care providers and patients to help improve the quality of care. In the spirit of contributing to the growth of public global goods, I-TECH and Medic have coordinated the application source code of the VMMC app. 
+
+### Users and hierarchy example
+
+{{< figure src="user_hierarchy.png"  link="user_hierarchy.png" alt="Users and hierarchy example" title="" class="left col-9">}}
+
+### VMMC direct-to-patient texting workflows
+
+#### 1. Patient automated check-in workflow to support patient care
+
+Eligible male clients with access to phones are registered on CHT to enable them receive daily check-in texts to help track the condition of the clients for 13 days after the circumcision. After the registration, the VMMC clients receive a registration text message confirming that they have enrolled in the text messaging follow up intervention. It is also possible for two household members belonging to the same household to be registered on CHT to be receiving texts using the same phone number.
+Clients registered on CHT receive preconfigured bidirectional daily follow up texts to check on their condition and clients respond to the check in texts by informing the Site Nurse whether they are experiencing any potential adverse effects. Clients who report potential adverse effects are followed up by facility Nurses and guided or referred to a health facility to be able to receive further care.
+{{< figure src="automated_check_in.png"  link="automated_check_in.png" alt="Patient automated check-in SMS workflows" title="" class="col-9">}}
```

Comment:
It might be a stretch, but let's see if we can use native HTML with images in a table.  Mocking this up with something like this:

```markdown
##### Patient automated check-in SMS workflows

| ![Enrollment](img.png) | ![SMS Check-in](img_1.png) | ![Referral](img_2.png) | ![Follow up](img_3.png) |
|-------|----------|---------|------|
| **1A** Enrollment form - Site nurse enrolls eligible VMMC clients in CHT using the enrollment form. | TK | TK  | TK|
```

I can get maybe close enough? Notice how hard to read the image based on is above it as compared to the native HTML one:

![image](https://user-images.githubusercontent.com/8253488/192845492-8c5bf301-3701-4792-b4cf-41c0858455f1.png)

Reply:
Also, please use the correct patient icon ([see here](https://github.com/medic/icon-library)) or reach out to @n-orlowski for help.  the patient icon used here is blurry.